### DISCUSSION TITLE
ANE Applovin

### DISCUSSION THREAD

**Discussion by jeuxdaily** [distriqt/ANE-IronSource#44] [General]
hello
I hope you find this message well
My question About ANE APPLOVIN
how can I  work only with distriqt applovin
if possible
thanks

**Comment by marchbold** [comment]
We don't have a dedicated AppLovin extension. We do have mediators for ironsource that will allow you to serve applovin ads through ironsource. 

Does that answer your question?

Cheers

**Comment by jeuxdaily** [comment]
hi 
thanks for your answer
what the method can show ads applovin in my apps

**Reply by marchbold** [reply]
You can add ironsource and the applovin mediator, then setup mediation through the console. 

See the docs for more info: https://docs.airnativeextensions.com/docs/ironsource/mediation/applovin

**Comment by jeuxdaily** [comment]
hi
I get this error 

---------------------------
Adobe Animate
---------------------------
Error creating files.

dx tool failed:Warning in C:\Users\xxxxx\AppData\Local\Temp\f689b754-5c2a-45b8-9d89-269c89933f31\play-services-clearcut-17.0.0.jar:com/google/android/gms/internal/clearcut/zzk.class:
  Type `java.lang.Long` was not found, it is required for default or static interface methods desugaring of `long com.google.android.gms.internal.clearcut.zzk.zza(byte[])`

Warning in C:\Users\xxxx\AppData\Local\Temp\f689b754-5c2a-45b8-9d89-269c89933f31\applovin-sdk-11.5.0.jar:com/applovin/exoplayer2/e/i/t.class:
  Type `java.lang.Math` was not found, it is required for default or static interface methods desugaring of `boolean com.applovin.exoplayer2.e.i.t.a(com.applovin.exoplayer2.l.y, byte[], int)`

Warning in C:\Users\xxxx\AppData\Local\Temp\f689b754-5c2a-45b8-9d89-269c89933f31\play-services-clearcut-17.0.0.jar:com/google/android/gms/internal/clearcut/zzk.class:
  Type `java.nio.ByteBuffer` was not found, it is required for default or static interface methods desugaring of `long com.google.android.gms.internal.clearcut.zzk.zzb(byte[], int)`

Warning in C:\Users\xxxxx\AppData\Local\Temp\f689b754-5c2a-45b8-9d89-269c89933f31\play-services-clearcut-17.0.0.jar:com/google/android/gms/internal/clearcut/zzge$zzc.class:
  Type `java.lang.Byte` was not found, it is required for default or static interface methods desugaring of `java.lang.Object com.google.android.gms.internal.clearcut.zzge$zzc.zza(int, java.lang.Object, java.lang.Object)`

Warning in C:\Users\xxxxx\AppData\Local\Temp\f689b754-5c2a-45b8-9d89-269c89933f31\play-services-ads-21.0.0.jar:com/google/android/gms/internal/ads/zzdon.class:
  Type `android.text.TextUtils` was not found, it is required for default or static interface methods desugaring of `void com.google.android.gms.internal.ads.zzdon.zza(java.lang.Object, java.util.Map)`

Warning in C:\Users\xxxxx\AppData\Local\Temp\f689b754-5c2a-45b8-9d89-269c89933f31\collection-1.1.0.jar:androidx/collection/SimpleArrayMap.class:
  Type `java.lang.System` was
---------------------------
OK   
---------------------------

**Reply by marchbold** [reply]
Are you using v11 of Java to package your application and do you have a recent Android SDK installed?

**Reply by jeuxdaily** [reply]
This problem when I publish my app with a package 33-1-1-406
but with the latest sdk version work fine
thanks
